<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoublePubStuChoiceItem</class>
 <widget class="QWidget" name="DoublePubStuChoiceItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VideoChoiceItem
{
	background-color: rgb(255, 255, 255);
	border-bottom-color:#E7EAEC;
	border-bottom-width: 1px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labIcone">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/common/img/mediaIc.png);
border-image: url(:/button/img/button/head.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:rgba(68,68,68,255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labEnableIco">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/idention/img/idention/Sigh.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="GifButton" name="btnChoice">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font:14px &quot;微软雅黑&quot;;
  color:rgb(68, 68, 68);
  border:1px solid rgb(221, 221, 221);
  background:rgb(255, 255, 255);
	  border-radius:2px;
}


QPushButton:hover{
	color: rgb(255, 255, 255);
	background:rgba(82,204,144,255);
	border:1px solid rgba(82,204,144,0);
}
QPushButton:pressed{
	color: rgb(68,68,68);
	border:1px solid rgb(221, 221, 221);
	background:rgb(255, 255, 255);
}
QPushButton:!enabled{
		color: rgba(82,204,144,255);
	border:0px solid rgb(255, 255, 255);
	background:rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GifButton</class>
   <extends>QPushButton</extends>
   <header>gifbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
